### PR TITLE
Update confirmed synonym evidence in Mondo

### PR DESCRIPTION
**Pull request by matentzn**
This is mostly to be able to review #8269.

The command being run to get this result is:

```
sh run.sh make update-synonyms-sync
```

(on #8269 branch)

### CODE REVIEW

**Review by joeflack4 — APPROVED** [review]
Approved. Left more comments in:
- #8269

**Comment by twhetzel** [comment]
Re-run the synonym sync pipeline to get fresh data then re-run this and confirm that it resolves all the qc errors.

**Comment by twhetzel** [comment]
@joeflack4 did you re-run the synonym sync pipeline?

File: src/ontology/mondo-edit.obo
[the remark this answers is not in the document]
Comment (joeflack4) [reply src/ontology/mondo-edit.obo:1]: - [ ] @joeflack4 to re-run pipeline
- [ ] Then, we re-look at QC failures

@twhetzel Check Slack; sent a message about my this etc. I'll respond here after the outputs are pushed.

[the remark this answers is not in the document]
Comment (joeflack4) [reply src/ontology/mondo-edit.obo:1]: @matentzn [New outputs pushed](https://github.com/monarch-initiative/mondo-ingest/commit/cf8356eacc476efa3ce04064678d5f86801e97b1). \*tosses ball\*

**Comment by matentzn** [comment]
Four errors left:

```
FAIL Rule ../sparql/qc/mondo/qc-exact-synonyms-non-exact-mappings.sparql: 4 violation(s)
entity,label,xref,synonym,code
http://purl.obolibrary.org/obo/MONDO_0012817,Ewing sarcoma,NCIT:C7542,PNET of Thoracopulmonary region,MONDO:relatedTo
http://purl.obolibrary.org/obo/MONDO_0020528,ACTH-dependent Cushing syndrome,ICD10CM:E[24](https://github.com/monarch-initiative/mondo/actions/runs/11776542244/job/32799138616?pr=8270#step:4:25).0,pituitary-dependent Cushing's disease,MONDO:directSiblingOf
http://purl.obolibrary.org/obo/MONDO_0004471,bacterial arthritis,NCIT:C26700,septic arthritis,MONDO:directSiblingOf
http://purl.obolibrary.org/obo/MONDO_000[25](https://github.com/monarch-initiative/mondo/actions/runs/11776542244/job/32799138616?pr=8270#step:4:26)14,hepatobiliary neoplasm,DOID:3117,hepatobiliary tumors,MONDO:mondoIsBroaderThanSource
```

In my opinion we need to independently:

(1) investigate why the synonym pipelines allows them to pass and (@joeflack4)
(2) fix them manually and merge this PR so we move on

File: src/ontology/mondo-edit.obo
[the remark this answers is not in the document]
Comment (matentzn) [reply src/ontology/mondo-edit.obo:1]: Receivwd and handled, see https://github.com/monarch-initiative/mondo/pull/8270#issuecomment-2467840373

**Comment by twhetzel** [comment]
Here's my ticket to fix them manually. I can make the changes, should we push to have another curator review the future PR this week during the curate-a-thon time period so we can close this by end of week?

**Comment by twhetzel** [comment]
It looks like the previous conflicts were fixed throughout the mondo curate-a-thon this week. I will close this PR and pull from fresh `master` and re-run `sh run.sh make update-synonyms-sync`

**Comment by twhetzel** [comment]
The issues were _not_ fixed during the curate-a-thon, I was looking at this incorrectly. The synonym sync is adding source information where it conflicts with an existing xref statement. In two cases, these xrefs did seem incorrect and there is a [PR to fix these](https://github.com/monarch-initiative/mondo/pull/8402).

However, in these two qc check failure cases, the xref does seem incorrect and should be addressed in the Synonym Sync:
- http://purl.obolibrary.org/obo/MONDO_0012817,Ewing sarcoma,NCIT:C7542,PNET of Thoracopulmonary region,MONDO:relatedTo
- http://purl.obolibrary.org/obo/MONDO_0002514,hepatobiliary neoplasm,DOID:3117,hepatobiliary tumors,MONDO:mondoIsBroaderThanSource

**Pull request closed without merging**